scripts/spack/devtools.json

```diff
@@ -3,9 +3,10 @@
 "package_version" : "fakeversion",
 "package_source_dir" : "../..",
 "package_final_phase" : "",
-"spack_url": "https://github.com/spack/spack",
-"spack_commit": "e75b76f433016a720027b41c0d36cdb9ef24a827",
+"spack_url": "https://github.com/adamjstewart/spack.git",
```

Comment:
This will hopefully get merged very soon as we can go back to the normal repo for the devtools.